bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -763,6 +758,13 @@ private void sendStateCommand(String channel, Command command) throws NanoleafEx
                     return;
                 }
                 break;
+            case CHANNEL_PANEL_LAYOUT:
+                @Nullable
+                Layout layout = controllerInfo.getPanelLayout().getLayout();
+                String layoutView = (layout != null) ? layout.getLayoutView() : "";
+                logger.info("Panel layout and ids for controller {} \n{}", thing.getUID(), layoutView);
```

Comment:
please remove this logging statement, or set to debug

Reply:
In this case it is exactly the intention to have it on "info" as it is the standard logging setting. It used to be info already before but was logged very often and filled the log. Now it is only logged upon request if needed intentionally as information for a standard user which is usually only done when you change the layout of your canvas. Hope that clarifies it :-)